### PR TITLE
Temporarily turn off BDC signed URLs [BT-236]

### PR DESCRIPTION
**Pull request by mcovarr**

### CODE REVIEW

File: martha/martha_v3.js
Comment (kshakir) [inline martha/martha_v3.js]: ToL: One can also comment out the variable?

File: test/martha/martha_v3.test.js
Comment (kshakir) [inline test/martha/martha_v3.test.js]: Same ToL: could be commented out

File: test/martha/martha_v3.test.js
Comment (kshakir) [inline test/martha/martha_v3.test.js]: FYI: There's no more "call 3", and I just noticed the sequence diagram doesn't have a link.

File: test/martha/martha_v3.test.js
Comment (kshakir) [inline test/martha/martha_v3.test.js]: Now that you're a certified Martha expert, what are your thoughts on these types of comments existing _only_ in this one test in the middle of the test suite?

File: test/martha/martha_v3.test.js
Comment (kshakir) [inline test/martha/martha_v3.test.js]: Would this be the same as:
```javascript
t.deepEqual({ ...result }, bdcDrsMarthaResult(googleSAKeyObject, null));
```

File: test/martha/martha_v3.test.js
Comment (kshakir) [inline test/martha/martha_v3.test.js]: Same replacing `drsAccessUrlResponse` with null.

File: test/martha/martha_v3.test.js
Comment (kshakir) [inline test/martha/martha_v3.test.js:845]: This test _really_ needs to be redone at some point. IMHO there's no reason to expect `drsResponse.access_methods is not iterable` to ever occur, so I think the error should not get special error handling.

For now, I'm fine leaving this and will continue to try and get this fixed in one of our upcoming PRs.

File: test/martha/martha_v3.test.js
Comment (kshakir) [inline test/martha/martha_v3.test.js:869]: I don't like leaving commented out code like this (search `ignore` in the Cromwell repo) but hopefully we'll be back soon 🤞

File: martha/martha_v3.js
Comment (mcovarr) [reply martha/martha_v3.js]: or delete it, whichever my reviewers like best 🙂

File: test/martha/martha_v3.test.js
Comment (mcovarr) [reply test/martha/martha_v3.test.js:845]: agreed this one was particularly gross

File: test/martha/martha_v3.test.js
Comment (mcovarr) [reply test/martha/martha_v3.test.js:869]: yeah I'm not imagining this as a "TODO" that never gets "TODONE" but as a PR that will be completely reverted in a week or two.

File: test/martha/martha_v3.test.js
Comment (mcovarr) [reply test/martha/martha_v3.test.js]: apparently so

File: test/martha/martha_v3.test.js
Comment (mcovarr) [reply test/martha/martha_v3.test.js]: this can probably go but since the whole PR will be reverted they may sneak back in 🙂

File: test/martha/martha_v3.test.js
Comment (mcovarr) [reply test/martha/martha_v3.test.js]: that link appears to be working for me...

Comment (kshakir) [reply test/martha/martha_v3.test.js]: 🤦

**Review by kshakir — APPROVED** [review]
Fine with the way it is

File: test/martha/martha_v3.test.js
Comment (breilly2) [inline test/martha/martha_v3.test.js:335]: This test is `martha_v3 calls returns the DRS name field for a file name even when it differs from the access url`. We can't really test that if we don't fetch an access URL. Maybe we can switch this test over to a hypothetical mock Kids First call?

File: test/martha/martha_v3.test.js
Comment (breilly2) [inline test/martha/martha_v3.test.js:235]: Not wrong, but maybe it would be more useful to switch this test over to use a DrsType and DRS response that will lead to an access URL being requested.

**Review by breilly2 — COMMENTED** [review]
A couple of comments. I think we need to modify a test or two to maintain current coverage.

File: test/martha/martha_v3.test.js
Comment (mcovarr) [reply test/martha/martha_v3.test.js:235]: There are Kids First tests including the access URL flow added in #212 which is only blocked on this PR

File: test/martha/martha_v3.test.js
Comment (mcovarr) [reply test/martha/martha_v3.test.js:335]: ~I believe this should also be covered with #212~ hmm "should be" but isn't... I will try adding that to #212

File: test/martha/martha_v3.test.js
Comment (breilly2) [reply test/martha/martha_v3.test.js:235]: Okay, as long as we reconcile this once both PRs are merged. Right now, this test is "martha_v3 doesn't call /access if the DRS response doesn't have an access_id, even if accessUrl is the only field requested from Martha".

File: test/martha/martha_v3.test.js
Comment (breilly2) [reply test/martha/martha_v3.test.js:335]: Okay, I understand how these 2 PRs are co-dependent. Not ideal, but it should work out once both are merged, and we can clean up later if needed.

**Review by breilly2 — APPROVED** [review]
👍   Looks good based on additional test coverage that is coming in #212.

**Comment by mcovarr** [comment]
jenkins retest

**Pull request merged**
